Sources/Common/Communication/Event.swift

```diff
@@ -195,3 +195,16 @@ public struct NewSubscriptionEvent: EventRepresentable {
         self.params = params
     }
 }
+
+/// When in-app SDK has fetched in-app messages from the server.
+public struct InAppMessagesFetchedEvent: EventRepresentable {
```

Comment:
I am OK with having the event in common or in-app. 

I put it here because right now, *all* of the eventbus events are in this file. So for organization purposes, I put it in here. 

I am happy to move it if you think that's best.

Reply:
Right now all the events in there are being utilized cross-module, as in track metrics, reset, screen etc. So this would be the only one relevant to `in-app` internal only, so my vote would be to move it there.